### DISCUSSION TITLE
[Feature Request]: Add single side border-color utilities

### DISCUSSION THREAD

**Discussion by rvrieling** [tailwindlabs/tailwindcss#1896] [Ideas]
I was wondering if it is a good idea to add a feature that has border side specific color. It is now not possible to add a color to the left border only. When you add a border color it will automatically apply to al borders and not specific. Please make this a new feature

**Comment by robbyrice** [comment]
I believe this already possible with the following code. It will add a 1px border to the left side of the div and the color of the left border will be teal.

You can take a look at theses links for some more examples.
https://tailwindcss.com/docs/border-width
https://tailwindcss.com/docs/border-color

```
<div class"border-l border-teal-500">
</div>
```

**Comment by luukdv** [comment]
These are some other alternatives: https://github.com/tailwindcss/discuss/issues/46#issuecomment-343185807

**Comment by carlos-duran** [comment]
I've read the old issue about this and the main concern was the amount of shipped classes. I think since we already have PurgeCSS inside Tailwind, this feature could be reconsidered.

**Comment by simonswiss** [comment]
Converting this to a `discussion` as this is for sure a recurring topic/request.
